scala support (#16)
pokey/vscode-parse-tree: README.md

2. Add a language to the dictionary at the top of `./src/extension.ts`
3. Add a reference to `onLanguage:yourlang` to the [activationEvents section of package.json](package.json). `yourlang` must be a [VSCode language identifier](https://code.visualstudio.com/docs/languages/identifiers).
4. Add your language to the top of the [Makefile](Makefile)

> 5. Hit `F5` in VSCode, with the Cursorless project open, to test your changes.

Author: Unsure about this. This is right for WSL, because we link it in. Regardless of this, you do also need to debug cursorless as well

Reviewer: This project is actually independent of cursorless, so I'm not sure I want to require users to install cursorless to test it.  To be fair, it's a bit tough to see if it's working without cursorless running, but you will definitely see if it crashes
```suggestion
5. Hit `F5` in VSCode, with this project open, to test your changes.
```

- Install the `pokey.command-server` extension on the host vscode
- Clone the extension in the WSL side.

> - If you're adding language support to `vscode-parse-tree`, you need to clone that as well, build it, and link it into the `vscode-server` extension folder: `ln -s ~/your/code/vscode-parse-tree ~/.vscode-server/extensions/parse-tree` for instance.

Author: I get this wrong literally every time I use `ln -s`, glad to see I'm not the only one

Reviewer: The way I remember it is that it's the same order as `mv` or `cp`: source to destination
